Makefile

```diff
@@ -2,7 +2,7 @@ cov-report = true
 
 
 lint:
-	flake8
+	pipenv run flake8
```

Comment:
This breaks the pipeline (https://travis-ci.org/argaen/aiocache/jobs/377003392#L483), you need to install pipenv in the syntax step too (tox file)

Reply:
I'll fix it soon